src/reanimated2/js-reanimated/MutableSet.js

```diff
@@ -0,0 +1,12 @@
+export default class MutableSet {
+  __mutableSet = true;
+  value = [];
+
+  add = (item) => {
+    this.value.push(item);
```

Comment:
Judging by name – if it's a set, then it shouldn't allow duplicate values? You can use [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) for it.